.github/workflows/build.yaml

```diff
@@ -121,7 +122,8 @@ jobs:
           retention-days: 7
 
       - name: Release
-        uses: qlik-oss/sn-gh-workflows/actions/semantic-release@v1
+        if: ${{ inputs.release }}
```

Comment:
Had some issues with dry-run for release-it. Workspaces actually also has a dry-run option. So what it did was do the dry run for the workspace command instead of a dry run of the release